src/components/DataTable/DataTable.tsx

```diff
@@ -36,6 +36,10 @@ export interface DataTableProps {
   showTotalsInFooter?: boolean;
   /** Lists of data points which map to table body rows. */
   rows: TableData[][];
+  /** Show a <Navigation /> when the data table is horizontally scrollable.
```

Comment:
```suggestion
  /** Whether or not to display column visibility and navigation buttons above the header when the table is wider than its container and collapses to be scrollable.
```

Reply:
Is this better?
"Display column visibility and navigation buttons above the header when the table horizontally collapses to be scrollable."

Reply:
I like what you have there @devonpmack 
I rewrote with teeny suggested edits:
`Display column visibility and navigation buttons above the header when the table collapses horizontally and is scrollable.`